models/response_model.go

```diff
@@ -0,0 +1,13 @@
+package models
+
+type PageDescription struct {
+	Size          int `json:"size"`
+	Number        int `json:"number"`
+	TotalElements int `json:"total_elements"`
+	TotalPages    int `json:"total_pages"`
```

Comment:
IMHO it is better to have caseing consistent, that's why I would prefer to have camel case here.

Reply:
Changed to camelCase in a different PR